jcl/src/java.base/share/classes/com/ibm/tools/attach/target/AttachHandler.java

```diff
@@ -50,7 +52,7 @@
 	/**
 	 * Time delay before we give up trying to terminate the wait loop.
 	 */
-	static final long shutdownTimeoutMs = Integer.getInteger("com.ibm.tools.attach.shutdown_timeout", 10000); //$NON-NLS-1$
+	static final long shutdownTimeoutMs = Integer.getInteger("com.ibm.tools.attach.shutdown_timeout", 10000).longValue(); //$NON-NLS-1$
```

Comment:
I'm not sure of the point of this change.

Reply:
It avoids a warning about auto-unboxing.